### ISSUE TITLE
Give radio fieldset a role of radiogroup

### ISSUE THREAD

**Issue by SiAdcock** [guardian/source#838]
**Is your feature request related to a problem? Please describe.**

By default, `<fieldset>` elements have a [role of `group`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/fieldset#technical_summary). Updating the `role` to `radiogroup` adds extra information that screen readers can convey to users.

**Comment by SiAdcock** [comment]
As mention in #902 there is an issue with eslint-plugin-jsx-a11y preventing us from making this change. It should have been fixed in jsx-eslint/eslint-plugin-jsx-a11y#746 but it doesn't appear to be live, or working?
